homeassistant/components/transmission/__init__.py

```diff
@@ -35,33 +33,41 @@
     entity_registry as er,
     selector,
 )
-from homeassistant.helpers.dispatcher import dispatcher_send
-from homeassistant.helpers.event import async_track_time_interval
 
 from .const import (
     ATTR_DELETE_DATA,
     ATTR_TORRENT,
     CONF_ENTRY_ID,
-    CONF_LIMIT,
-    CONF_ORDER,
-    DATA_UPDATED,
     DEFAULT_DELETE_DATA,
-    DEFAULT_LIMIT,
-    DEFAULT_ORDER,
-    DEFAULT_SCAN_INTERVAL,
     DOMAIN,
-    EVENT_DOWNLOADED_TORRENT,
-    EVENT_REMOVED_TORRENT,
-    EVENT_STARTED_TORRENT,
     SERVICE_ADD_TORRENT,
     SERVICE_REMOVE_TORRENT,
     SERVICE_START_TORRENT,
     SERVICE_STOP_TORRENT,
 )
+from .coordinator import TransmissionDataUpdateCoordinator
 from .errors import AuthenticationError, CannotConnect, UnknownError
 
 _LOGGER = logging.getLogger(__name__)
 
+CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
+
+PLATFORMS = [Platform.SENSOR, Platform.SWITCH]
+
+MIGRATION_NAME_TO_KEY = {
```

Comment:
Doesn't seem to be used?

Reply:
This is used in the `update_unique_id` method in `async_setup_entry`.